tests/test_bar_data.py

```diff
@@ -522,158 +525,145 @@ def test_overnight_adjustments(self):
                 self.assertEqual(value, expected[field])
 
 
-class TestDailyBarData(TestBarDataBase):
-    @classmethod
-    def setUpClass(cls):
-        cls.tempdir = TempDirectory()
-
-        # asset1 has a daily data for each day (1/5, 1/6, 1/7)
-        # asset2 only has daily data for day2 (1/6)
-
-        cls.env = TradingEnvironment()
-
-        cls.days = cls.env.days_in_range(
-            start=pd.Timestamp("2016-01-05", tz='UTC'),
-            end=pd.Timestamp("2016-01-08", tz='UTC')
-        )
-
-        cls.env.write_data(equities_data={
-            sid: {
-                'start_date': cls.days[0],
-                'end_date': cls.days[-1],
-                'symbol': "ASSET{0}".format(sid)
-            } for sid in [1, 2, 3, 4, 5, 6, 7, 8]
-        })
-
-        cls.ASSET1 = cls.env.asset_finder.retrieve_asset(1)
-        cls.ASSET2 = cls.env.asset_finder.retrieve_asset(2)
-        cls.SPLIT_ASSET = cls.env.asset_finder.retrieve_asset(3)
-        cls.ILLIQUID_SPLIT_ASSET = cls.env.asset_finder.retrieve_asset(4)
-        cls.MERGER_ASSET = cls.env.asset_finder.retrieve_asset(5)
-        cls.ILLIQUID_MERGER_ASSET = cls.env.asset_finder.retrieve_asset(6)
-        cls.DIVIDEND_ASSET = cls.env.asset_finder.retrieve_asset(7)
-        cls.ILLIQUID_DIVIDEND_ASSET = cls.env.asset_finder.retrieve_asset(8)
-        cls.ASSETS = [cls.ASSET1, cls.ASSET2]
+class TestDailyBarData(WithBarDataChecks,
+                       WithDataPortal,
+                       ZiplineTestCase):
+    SIM_PARAMS_START = pd.Timestamp('2016-01-05', tz='UTC')
+    SIM_PARAMS_END = pd.Timestamp('2016-01-08', tz='UTC')
 
-        cls.adjustments_reader = cls.create_adjustments_reader()
-        cls.data_portal = DataPortal(
-            cls.env,
-            equity_daily_reader=cls.build_daily_data(),
-            adjustment_reader=cls.adjustments_reader
-        )
+    sids = set(range(1, 9))
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.tempdir.cleanup()
+    SPLIT_ASSET_SID = 3
+    ILLIQUID_SPLIT_ASSET_SID = 4
+    MERGER_ASSET_SID = 5
+    ILLIQUID_MERGER_ASSET_SID = 6
+    DIVIDEND_ASSET_SID = 7
+    ILLIQUID_DIVIDEND_ASSET_SID = 8
 
     @classmethod
-    def create_adjustments_reader(cls):
-        path = cls.tempdir.getpath("test_adjustments.db")
-
-        adj_writer = SQLiteAdjustmentWriter(
-            path,
-            cls.env.trading_days,
-            MockDailyBarReader()
+    def make_equity_info(cls):
+        return pd.DataFrame.from_dict(
```

Comment:
Is there a reason this isn't just `make_simple_equity_info` with a symbols list?